src/modules/navigator/mission_feasibility_checker.cpp

```diff
@@ -343,6 +343,15 @@ MissionFeasibilityChecker::checkTakeoff(const mission_s &mission, float home_alt
 				return false;
 			}
 
+			// get the parameter for min take off altitude in meters
+			float takeoff_alt_min = _navigator->get_takeoff_min_alt();
+
+			// check if it is set to something meaningful and if the take off altitude is set above the parameter
+			if (takeoff_alt_min > 0 && takeoff_alt < takeoff_alt_min) {
```

Comment:
Shouldn't `MIS_TAKEOFF_ALT` only apply relative to the home altitude? That's already happening if the mission item is relative, but if it's GPS altitude I think the minimum altitude should then be HOME + MIS_TAKEOFF_ALT.